### ISSUE TITLE
[Bug]: No one is thanking Mg Meyers Enough for This Awesome Plugin

### ISSUE THREAD

**Issue by skillhacker-code** [mgmeyers/obsidian-kanban#351]
### Describe the bug

Mg Meyers Created an awesome plugin and no one is thanking him enough 

### Expected Behavior

Everyone Should Thank him , Those who can donate should do so 

### Steps to reproduce

1. Reply to this bug 
2. Thank Mg Meyers
3. Sponsor Him https://github.com/mgmeyers
4. Buy Him a Coffee https://www.buymeacoffee.com/mgme 

THANK YOU SO MUCH MATHEW, YOU HAVE DONE A WONDERFUL JOB. 

### Operating system

Windows

**Comment by htmlbanjo** [comment]
+1, this really takes Obsidian from an accessible way to spin up a personal wiki and turns it into something that can be an easy Jira alternative for people who aren't to the level they need JQL etc.

for personal projects this has been sorely lacking: a nice all-in-one where I can have something Trello-like along with my documentation w/o having to resort to the cloud.

**Comment by keenfoong** [comment]
Indeed, I use the kanban plugin every day for work (typical todo doing and done) and for personal things (saving and clipping images and artwork from the internet). It's an invaluable addition!

**Comment by perkam** [comment]
@skillhacker-code is right. This plugin is great. Thank you @mgmeyers :)

**Comment by dcoales** [comment]
Agreed, the plugin is awesome.

**Comment by TfTHacker** [comment]
Ditto Ditto Ditto

We can thank him by contributing to the project. There is a link for that right on the repository.

Kind words and a few coffees convey our feelings nicely

**Comment by JJger148** [comment]
I love this bug report! :D 
But it is so true!
Thank you very much indeed mgmeyers for your awesome work!

**Comment by SuperTempAccount** [comment]
Thank you Mg Meyers 🙏 You've made my life a little easier and my days a little happier with this simple plugin ❤
